### DISCUSSION TITLE
阿里云盘OPEN每15分钟需要刷新当前页面才能继续跳转播放

### DISCUSSION THREAD

**Discussion by yanzhang4013** [AlistGo/alist#5614] [General]
刚更新了3.29.1版本，发现有了这个问题，找网上找解决方案，https://tieba.baidu.com/p/8747150411?pn=1，但是感觉又没有给出来一个有效的解决方法，请问大家都有这个情况吗？

**Comment by citylyz** [comment]
用网页界面的阿里云盘默认播放器aliyun video previewer，没有限速，如果选video模式，就是限速了，只有500KB。感觉是用非阿里云播放器限速了。

**Comment by anwen-anyi** [comment]
目前使用`aliyun video`播放器只能播放15分钟，等待后续优化。
